react-components/src/architecture/concrete/pointsOfInterest/types.ts

```diff
@@ -28,7 +28,7 @@ export type PointOfInterest<IdType> = {
 export function createPointsOfInterestPropertiesFromPointAndTitle(
   point: Vector3,
   scene: DmsUniqueIdentifier,
-  contents: ReactNode[],
+  contents: string[],
   associatedInstance: InstanceReference | undefined
```

Comment:
I think want this function to take the `title` + `description` explicitly in their own parameters. Right now it's a bit cryptic what the `contents`-argument means :sweat_smile:

Reply:
yes sure
https://github.com/cognitedata/reveal/pull/4929/commits/c0bebe9a063de9f6d742f5648af6bf4636363e34